### DISCUSSION TITLE
Clear Step by Step guide for Glances, influxdb and Grafana

### DISCUSSION THREAD

**Discussion by jslegers73** [nicolargo/glances#2464] [Q&A]
Hello,

Is there a clear step by step guide on how to setup Glances with Grafana. I have tried this from Nico : https://github.com/nicolargo/docker-influxdb2-grafana-glances but then I still have to setup Grafana. 

Sorry for this noob question but I can't get it to work. 

John

**Comment by BenRoe** [comment]
1. Open the Grafana WebUI
2. If it's the first login us admin admin
3. [Setup your influxdb under `Home - Connections - Data sources` ](https://www.stackhero.io/en/services/Grafana/documentations/Getting-started#connecting-grafana-to-influxdb-v2)
4. Click left top on Home and then Dashboard
5. Then blue button on the right `New` and Import
6. Open [Glances Grafana Import JSON](https://raw.githubusercontent.com/nicolargo/glances/develop/conf/glances-grafana-flux.json) (InfluxDB2) and copy the complete text
7. Paste it into `Import via panel json` in Grafana
